modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-test/src/testIntegration/java/com/liferay/headless/commerce/admin/pricing/resource/v2_0/test/DiscountAccountGroupResourceTest.java

```diff
@@ -175,6 +175,12 @@ public void testGetDiscountIdDiscountAccountGroupsPageWithFilterDateTimeEquals()
 		throws Exception {
 	}
 
+	@Override
+	@Test
+	public void testGetDiscountIdDiscountAccountGroupsPageWithFilterDoubleEquals()
+		throws Exception {
+	}
+
```

Comment:
I don't know why it is necessary to do this (I've seen that this is common in other files of commerce). So, I don't know if this is our responsibility or we should involve the commerce team to check this.
For me is weird to override a test for an empty test, maybe is my lack of knowledge 😅

Reply:
The overwritten method is included to avoid the test from the parent. The point is the following code from the parent entityModelResource is null, so it throws a NullPointerException:

```
EntityModelResource entityModelResource =
    (EntityModelResource)_discountAccountGroupResource;

EntityModel entityModel = entityModelResource.getEntityModel(
    new MultivaluedHashMap());
```